### ISSUE TITLE
latest release (2.0.15) broken?

### ISSUE THREAD

**Issue by wuestkamp** [gehrj/vue-country-region-select#53]
release 2.0.15 seems to be broken and I get build errors:

```
This dependency was not found:
* vue-country-region-select in ./src/main.js
```

node_modules/vue-country-region-select contains just a README

2.0.14 works

**Comment by seekingperfection** [comment]
Same for me, how to get 2.0.14 ?  @wuestkamp

**Comment by wuestkamp** [comment]
add this into your `package.json`
`"vue-country-region-select": "2.0.14"`
then `npm install`

**Comment by seekingperfection** [comment]
still same problem -_-

2021-05-16, sk, 17:27 Kim Wuestkamp ***@***.***> rašė:

**Comment by seekingperfection** [comment]
Thanks man, appreciate it <3

**Comment by gehrj** [comment]
hey thanks there was a mistake in the build, this should be fixed in 2.0.16 now!
